app/models/allocated_offender.rb

```diff
@@ -19,7 +19,7 @@ def initialize(staff_id, allocation, offender)
   end
 
   def latest_movement_date
```

Comment:
Could this also be renamed to include temp?

```suggestion
  def latest_temp_movement_date
```

Reply:
In fact – presumably the next PR will be to relocate this method (into OffenderBase) or to use AllocatedOffender more widely. So presumably this isn't going to be hanging around for long..?